gtda/mapper/visualization.py

```diff
@@ -226,8 +225,9 @@ def plot_interactive_mapper_graph(pipeline, data, layout='kamada_kawai',
 
     Returns
     -------
-    display : :class:`DisplayHandle` object
-        Displays the interactive Mapper graph widget.
+    fig : :class:`ipywidgets.VBox` object. A widget, that contains
```

Comment:
Maybe let's call this ``box`` -- matches the actual code too.

Reply:
This is quite difficult to explain, but how about "a box containing the following widgets: parameters of the clustering algorithm, parameters for the covering scheme, a Mapper graph arising from those parameters, a validation box, and logs"